charts/project-origin-registry/run_kind_test.sh

```diff
@@ -90,10 +93,10 @@ cat << EOF > "${wallet_filename}"
 service:
   type: NodePort
   nodePort: ${wallet_nodeport}
-wallet:
+config:
   externalUrl: http://localhost:${wallet_nodeport}
-redis:
-  enabled: true
+  jwt:
+    allowAnyJwtToken: true
```

Comment:
Should I be worried about this?

Reply:
This is only in the test, as not to have to mess with it. 
This is a test of the registries, and not the wallet, just using the wallet system for the test.